collections/Douce/MS_Douce_311.xml

```diff
@@ -30,61 +30,559 @@
+                           <locus>(fol. 118r)</locus>
+                           <title type="desc">St Martha</title>
+                        </msItem>
+                        <msItem>
+                           <locus>(fols. 118v–119r)</locus>
+                           <title type="desc">St Catherine</title>
+                        </msItem>
+                        <msItem>
+                           <locus>(fol. 119r–v)</locus>
+                           <title type="desc">St Barbara</title>
+                        </msItem>
+                        <msItem>
+                           <locus>(fol. 119v)</locus>
+                           <title type="desc">St Margaret</title>
+                        </msItem>
+                        <msItem>
+                           <locus>(fols. 119v–120a verso)</locus>
+                           <title type="desc">St Apollonia</title>
+                        </msItem>
+                        <msItem>
+                           <locus>(fols. 120a verso–120d verso)</locus>
+                           <note>Blank, ruled</note>
+                        </msItem>
+                     </msItem>
+                     <msItem>
+                        <locus>(fol. 121r–v)</locus>
+                        <title type="desc">Short prayers</title>
+                        <textLang mainLang="la" otherLangs="frm">Rubrics in Middle French</textLang>
+                     </msItem>
+                     <msItem>
+                        <title type="desc">Prayers to Christ</title>
+                        <msItem>
+                           <locus>(fol. 122r)</locus>
+                           <note>Blank</note>
+                        </msItem>
+                        <msItem>
+                           <locus>(fols. 122v–124r)</locus>
+                           <title type="desc">Prayer to Christ</title>
+                           <textLang mainLang="frm">Middle French</textLang>
+                        </msItem>
+                        <msItem>
+                           <locus>(fol. 124a verso)</locus>
+                           <title type="desc">Prayer to Christ</title>
+                           <textLang mainLang="la">Latin</textLang>
+                        </msItem>
+                        <msItem>
+                           <locus>(fols. 124b verso-124c verso)</locus>
+                           <note>Blank, ruled</note>
+                        </msItem>
+                     </msItem>
+                     <msItem>
+                        <title type="desc">Office of the dead</title>
+                        <msItem>
+                           <locus>(fol. 125r)</locus>
+                           <note>Blank</note>
+                        </msItem>
+                        <msItem>
+                           <locus>(fols. 125v–145r)</locus>
+                           <title key="work_13574" type="desc">Office of the dead, Use of Rome</title>
```

Comment:
we don't usually provide a key attribute where the Office occurs as part of a book of hours, see the usage note at work_13574